lib/responses/json.js

```diff
@@ -0,0 +1,149 @@
+'use strict';
+/**
+ * Send JSON Response
+ *
+ * Usage:
+ * return res.json(statusCode);
+ * return res.json(statusCode, data);
+ *
+ * @param  {Object} data
+ */
+
+/**
+ * res.json() changed order of arguments a few times.
+ * Sails.js /lib/router/res.js provides a private function to normalize the arguments.
+ * https://github.com/balderdashy/sails/blob/0506f0681590dc92986985bc39609c88b718a997/lib/router/res.js#L329-L360
+ */
+/* eslint-disable */
```

Comment:
Don't want to change the function from Sails.js to keep it maintainable and there are some eslint issues in Sails.js code. E.q. the else will fail due to [1tbs rule](http://eslint.org/docs/rules/brace-style#tbs).